docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss

```diff
@@ -125,6 +125,15 @@
     }
   }
 
+  @include hb-themes(('traditional', 'warbler')) {
+    @include hb-pairing-color('color', 'secondary');
+
+    &:hover,
+    &:focus {
+      @include hb-pairing-color('color', 'tertiary-darken-20');
```

Comment:
I believe this line should be `@include hb-pairing-color('color', 'tertiary');` according to the ticket AC. Double check this though.

Reply:
```suggestion
      @include hb-pairing-color('color', 'tertiary');
```